docs/9.0/api/manualRowMove.md

```diff
@@ -88,12 +88,12 @@ _manualRowMove.dragRow(row, dropIndex) ⇒ boolean_
 
 Drag a single row to drop index position.
 
-**Emits**: [`Hooks#event:beforeRowMove`](./hooks/#beforeRowMove), [`Hooks#event:afterRowMove`](./hooks/#afterRowMove)  
+**Emits**: [`Hooks#event:beforeRowMove`](@/api/pluginHooks.md#beforerowmove), [`Hooks#event:afterRowMove`](@/api/pluginHooks.md#afterrowmove)  
 
 | Param | Type | Description |
 | --- | --- | --- |
 | row | `number` | Visual row index to be dragged. |
-| dropIndex | `number` | Visual row index, being a drop index for the moved rows. Points to where we are going to drop the moved elements. To check visualization of drop index please take a look at [documentation](@/guides/rows/row-moving.md#drag-and-move-actions-of-manualrowmove-plugin). |
+| dropIndex | `number` | Visual row index, being a drop index for the moved rows. Points to where we are going to drop the moved elements. To check visualization of drop index please take a look at [documentation](@/guides/rows/row-moving.md#dragandmoveactionsofmanualrowmoveplugin). |
```

Comment:
`#drag-and-move-actions-of-manualrowmove-plugin`

Reply:
#8440